docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-con.adoc

```diff
@@ -0,0 +1,13 @@
+[id='environment-authoring-con']
+= Authoring environment
+You can deploy an environment for creating and modifying processes using {CENTRAL}. It consists {CENTRAL} for the authoring work and {KIE_SERVER} for test execution of the processes.
+
+Depending on your needs, you can deploy either a single authoring environment of a high-availability (HA) authoring environment. 
+
+A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. The {KIE_SERVER} includes an embedded H2 database engine. This type of environment uses the least possible amount of resources.
+
+image::Authoring.png[Architecture diagram of a single authoring infrastructure]
+
+A HA authoring environment contains several pods. Both {CENTRAL} and {KIE_SERVER} are provided in scalable pods that can run in parallel and share persistent storage. The database is provided by a separate high-availability service. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are onvolved in authoring at the same time.
```

Comment:
A HA authoring environment -> An HA authoring environment
and share persistent storage: clarify that only Business Central pods share persistent storage
onvolved -> involved